packages/camera/camera/lib/camera.dart

```diff
@@ -323,30 +209,27 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
     try {
       _creatingCompleter = Completer<void>();
-      final Map<String, dynamic> reply =
-          await _channel.invokeMapMethod<String, dynamic>(
-        'initialize',
-        <String, dynamic>{
-          'cameraName': description.name,
-          'resolutionPreset': serializeResolutionPreset(resolutionPreset),
-          'enableAudio': enableAudio,
-        },
-      );
-      _textureId = reply['textureId'];
+
+      _cameraId = await CameraPlatform.instance.initializeCamera(
+          description, resolutionPreset,
+          enableAudio: enableAudio);
```

Comment:
nit: correct formatting

```suggestion
      _cameraId = await CameraPlatform.instance.initializeCamera(
          description, 
          resolutionPreset,
          enableAudio: enableAudio,
      );
```